frontend/src/features/idir/search/components/IDIRSearchResults.tsx

```diff
@@ -123,11 +122,13 @@ export const IDIRSearchResults = memo(
               </Box>
             );
           }}
+          initialState={{
+            sorting: [{ id: "permitIssueDateTime", desc: true }],
+          }}
```

Comment:
setting `state` breaks the column sorting, but need the `initialState` for the spec. https://github.com/bcgov/onRouteBCSpecification/blob/main/EO%20Dashboard/EO%20Search%20for%20Permits.feature#L24